modules/building/pages/apply-task-migrations.adoc

```diff
@@ -0,0 +1,113 @@
+migrations by themselves, this document illustrates how to do that.
+
+== Apply migration with pipeline-migraiton-tool
+
+. Install pipeline-migration-tool:
+
++
+Choose a release from
+https://github.com/konflux-ci/pipeline-migration-tool/releases[releases] page
+and install a specific version.
+
++
+[source,bash]
+----
+pipx install https://github.com/konflux-ci/pipeline-migration-tool/archive/refs/tags/v0.4.2.zip
+----
+
++
+This installs two executables in `$PATH` - `pipeline-migration-tool` and its alias, `pmt`.
+In this document, we will use `pmt`.
+
+. Identify bundle upgrades
+
++
+If there is some migration, you can see "migration" appearing in the Notes
+column in the description of a Renovate update pull request. For example:
+
++
+image::doc-mintmaker-renovate-update-pr.png[]
+
++
+Then, check the bundle update from changes page:
+
++
+[source,diff]
+----
+           - name: name
+             value: clamav-scan
+           - name: bundle
+-            value: quay.io/konflux-ci/tekton-catalog/task-clamav-scan:0.2@sha256:7749146f7e4fe530846f1b15c9366178ec9f44776ef1922a60d3e7e2b8c6426b
++            value: quay.io/konflux-ci/tekton-catalog/task-clamav-scan:0.3@sha256:cce2dfcc5bd6e91ee54aacdadad523b013eeae5cdaa7f6a4624b8cbcc040f439
+           - name: kind
+             value: task
+         resolver: bundles
+----
+
+. Run command to apply migration. Convert the bundle update into upgrade data,
+  then run CLI `pmt` from the Component repository root directory with the
+  upgrade data:
+
++
+[source,bash]
+----
+cd path/to/component-repo
+
+pmt migrate -u '
+[
+    {
+        "depName": "quay.io/konflux-ci/tekton-catalog/task-clamav-scan",
+        "currentValue": "0.2",
+        "currentDigest": "sha256:7749146f7e4fe530846f1b15c9366178ec9f44776ef1922a60d3e7e2b8c6426b",
+        "newValue": "0.3",
+        "newDigest": "sha256:cce2dfcc5bd6e91ee54aacdadad523b013eeae5cdaa7f6a4624b8cbcc040f439",
+        "packageFile": ".tekton/<pipeline file>", <.>
+        "parentDir": ".tekton",
+        "depTypes": ["tekton-bundle"]
+    },
+    {
+        "depName": "quay.io/konflux-ci/tekton-catalog/task-clamav-scan",
+        "currentValue": "0.2",
+        "currentDigest": "sha256:7749146f7e4fe530846f1b15c9366178ec9f44776ef1922a60d3e7e2b8c6426b",
+        "newValue": "0.3",
+        "newDigest": "sha256:cce2dfcc5bd6e91ee54aacdadad523b013eeae5cdaa7f6a4624b8cbcc040f439",
+        "packageFile": ".tekton/<pipeline file>", <.>
+        "parentDir": ".tekton",
+        "depTypes": ["tekton-bundle"]
+    }
+    , ... <.>
+]
+'
+----
+
++
+<.> Replace `<pipeline file>` with actual file name which is under `.tekton/` directory.
+<.> Likewise. Specify another pipeline file.
+<.> You can specify more upgrades if there are any.
+
++
+[NOTE]
+====
+When saying build pipeline and pipeline file, it means a YAML file including
+tekton `Pipeline` definition or `PipelineRun` definition where `Pipeline` is
+embedded in `.spec.pipelineSpec`.
```

Comment:
```suggestion
In this context, 'build pipeline' or 'pipeline file' are refering to a YAML file including a
Tekton `Pipeline` or `PipelineRun` definition.
```

I feel like explaining what a pipelineRun is here is confusing. I would refer users to https://tekton.dev/docs/concepts/concept-model/#steps-tasks-and-pipelines as a last resort.